tests/unit/restapi/lib/actions.py

```diff
@@ -101,3 +104,32 @@ def register_queue(
         json=payload,
         follow_redirects=True,
     )
+
+
+def create_plugin_parameter_type(
+    client: FlaskClient,
+    name: str,
+    group_id: int,
+    structure: dict[str, Any] | None = None,
```

Comment:
> should have a `description` as well. Present in ORM and added to schema by #460

```suggestion
    client: FlaskClient,
    name: str,
    group_id: int,
    structure: dict[str, Any] | None = None,
    description: str | None = None,
```
Adding description as mentioned.